README.md

```diff
@@ -1,3 +1,7 @@
+<div align="center" style="padding: 30px 0px 20px 0px;">
```

Comment:
I'm not sure if the `30px` on top looks good. Probably `20px` makes it even between top and bottom.